integration-cli/docker_cli_events_test.go

```diff
@@ -78,9 +78,13 @@ func (s *DockerSuite) TestEventsUntag(c *check.C) {
 }
 
 func (s *DockerSuite) TestEventsLimit(c *check.C) {
-	// Limit to 8 goroutines creating containers in order to prevent timeouts
+	// Limit to 4 goroutines creating containers in order to prevent timeouts
```

Comment:
@darrenstahlmsft this comment looks not true, we still limiting to 8 everywhere apart from Windows.